tests/baselines/reference/tsserver/resolutionCache/not-sharing-across-references.js

```diff
@@ -0,0 +1,182 @@
+Info 46   [00:01:15.000] File '/src/projects/node_modules/moduleX/index.tsx' does not exist.
+Info 47   [00:01:16.000] File '/src/projects/node_modules/moduleX/index.d.ts' exist - use it as a name resolution result.
+Info 48   [00:01:17.000] Resolving real path for '/src/projects/node_modules/moduleX/index.d.ts', result '/src/projects/node_modules/moduleX/index.d.ts'.
+Info 49   [00:01:18.000] ======== Module name 'moduleX' was successfully resolved to '/src/projects/node_modules/moduleX/index.d.ts'. ========
+Info 50   [00:01:19.000] DirectoryWatcher:: Added:: WatchInfo: /src/projects/app/node_modules 1 undefined Project: /src/projects/app/tsconfig.json WatchType: Failed Lookup Locations
+Info 51   [00:01:20.000] Elapsed:: *ms DirectoryWatcher:: Added:: WatchInfo: /src/projects/app/node_modules 1 undefined Project: /src/projects/app/tsconfig.json WatchType: Failed Lookup Locations
+Info 52   [00:01:21.000] FileWatcher:: Added:: WatchInfo: /a/lib/lib.d.ts 500 undefined Project: /src/projects/app/tsconfig.json WatchType: Missing file
+Info 53   [00:01:22.000] Finishing updateGraphWorker: Project: /src/projects/app/tsconfig.json Version: 1 structureChanged: true structureIsReused:: Not Elapsed:: *ms
+Info 54   [00:01:23.000] Project '/src/projects/app/tsconfig.json' (Configured)
+Info 55   [00:01:24.000] 	Files (4)
+	/src/projects/node_modules/moduleX/index.d.ts
+	/src/projects/app/appA.ts
+	/src/projects/common/moduleB.ts
+	/src/projects/app/appB.ts
+
+
+	../node_modules/moduleX/index.d.ts
+	  Imported via "moduleX" from file 'appA.ts'
+	  Imported via "moduleX" from file '../common/moduleB.ts'
+	appA.ts
+	  Matched by default include pattern '**/*'
+	../common/moduleB.ts
+	  Imported via "../common/moduleB" from file 'appB.ts'
+	appB.ts
+	  Matched by default include pattern '**/*'
+
+Info 56   [00:01:25.000] -----------------------------------------------
+Info 57   [00:01:26.000] Search path: /src/projects/app
+Info 58   [00:01:27.000] For info: /src/projects/app/tsconfig.json :: No config files found.
+Info 59   [00:01:28.000] Project '/src/projects/app/tsconfig.json' (Configured)
+Info 59   [00:01:29.000] 	Files (4)
+
+Info 59   [00:01:30.000] -----------------------------------------------
+Info 59   [00:01:31.000] Open files: 
+Info 59   [00:01:32.000] 	FileName: /src/projects/app/appB.ts ProjectRootPath: undefined
+Info 59   [00:01:33.000] 		Projects: /src/projects/app/tsconfig.json
+After request
+
+PolledWatches::
+/src/projects/app/node_modules:
+  {"pollingInterval":500}
+/a/lib/lib.d.ts:
+  {"pollingInterval":500}
+
+FsWatches::
+/src/projects/app/tsconfig.json:
+  {}
+/src/projects/app/appa.ts:
+  {}
+/src/projects/common/tsconfig.json:
+  {}
+/src/projects/common/moduleb.ts:
+  {}
+
+FsWatchesRecursive::
+/src/projects/app:
+  {}
+/src/projects/common:
+  {}
+/src/projects/node_modules:
+  {}
+
+Info 59   [00:01:34.000] response:
+    {
+      "responseRequired": false
+    }
```

Comment:
I can’t figure out where the relevant part of these baselines is. Is it possible to write the contrasting sharing/not-sharing tests in a way that asserts only the relevant difference instead of creating two very large baselines that are nearly identical?

Reply:
Agreed diff would be great but that’s not easy with current baselines and traces .. so tried to keep it same as tsserver log that we anyways use for debugging user issues too ..